README.md

```diff
@@ -7,8 +7,8 @@ Kubernetes Cluster.
 
 The following packages are included in Fury Kubernetes OPA module:
 
-- [Gatekeeper](katalog/gatekeeper): Ready to use gatekeeper deployment plus a set of rules. Version: **v3.1.0-beta.8**
-  - [Gatekeeper Core](katalog/gatekeeper/core): Gatekeeper deployment, ready to apply rules. Version: **v3.1.0-beta.8**
+- [Gatekeeper](katalog/gatekeeper): Ready to use gatekeeper deployment plus a set of rules. Version: **v3.1.0-beta.9**
```

Comment:
Sure we want to support betas?

Reply:
No, but we don't have any other alternative. Even on beta is the only version stable from OPA ;)